tests/e2e/tests/devfiles/Go.spec.ts

```diff
@@ -46,7 +46,7 @@ suite(`${workspaceStack} test`, async () => {
         projectManager.openFile(fileFolderPath, fileName);
     });
 
-    suite('Test golang example', async () => {
+    suite.skip('Test golang example', async () => { // test always fails, because the task does not show a notification about return code
```

Comment:
I'm not sure about skipping this test. In 7.19.x the notification was removed in all stacks so we would need to switch to another approach. I would leave it unskipped.
issue for fix: https://github.com/eclipse/che/issues/17973#event-3823255291
upstream PR with the change: https://github.com/eclipse-theia/theia/pull/8331
blocker for implementing the fix: https://github.com/eclipse/che/issues/17978

Reply:
but if it will be unskipped, it will be causing test failures

Reply:
Well, but then it means that it should be skipped in all the devfile tests - I don't see much value in skipping that just in one of them.

Reply:
I will test it on latest che and skip where necessary